lib/modules/manager/gleam/artifacts.ts

```diff
@@ -0,0 +1,89 @@
+import is from '@sindresorhus/is';
```

Comment:
ignore this! copied from bun, was hoping to adapt the manifest updater using a gleam docker image. i don't think we have a gleam docker image publishing yet? i see one from a few years ago